cpp/CMakeLists.txt

```diff
@@ -201,7 +161,7 @@ if(BUILD_TESTS)
     if(GTEST_FOUND)
         message(STATUS "Google C++ Testing Framework (Google Test) found in ${GTEST_ROOT}")
         include_directories(${GTEST_INCLUDE_DIR})
-        add_subdirectory(${CMAKE_SOURCE_DIR}/tests)
+        # add_subdirectory(${CMAKE_SOURCE_DIR}/tests)
```

Comment:
I don't understand this change. Is commenting out tests meant to be temporary?

Reply:
This is a temporary change. Currently, there is a cyclical loop whereby NVStrings needs GTest, and the `cudf/cpp/tests` needs NVStrings.

I don’t like the idea of splitting the GTest logical block and the compiling of the tests; so, I’m playing with a few ideas which integrate the dependency chains between NVStrings and cuDF to avoid the need to split altogether. Essentially, having NVStrings and cuDF depend on GTests/GBench/etc. only once.